### ISSUE TITLE
Issues with reek

### ISSUE THREAD

**Issue by mbajur** [AdWerx/pronto-ruby#8]
Hey there!

I'm trying to add a reek step to pronto action but it gives me:

```
/usr/local/lib/ruby/site_ruby/2.5.0/rubygems/specification.rb:2302:in `raise_if_conflicts': Unable to activate reek-5.5.0, because parser-2.7.0.1 conflicts with parser (>= 2.5.0.0, != 2.5.1.1, < 2.7) (Gem::ConflictError)
```

You can check that here: https://github.com/mbajur/pronto-ruby-bug/pull/2/checks?check_run_id=372623270

Here is the repro PR you're a contributor in if needed: https://github.com/mbajur/pronto-ruby-bug/pull/2

Thanks!

**Comment by jbielick** [comment]
@mbajur Thanks for working with me on these. I think I released a bad docker image for this so `2.3.1` should work. [I made a commit on that PR and it looks like it ran okay](https://github.com/mbajur/pronto-ruby-bug/runs/374577186)

**Comment by mbajur** [comment]
Sweet, thank you!
